snap/info.go

```diff
@@ -32,6 +36,8 @@ type Info struct {
 	Apps        map[string]*AppInfo
 	Plugs       map[string]*PlugInfo
 	Slots       map[string]*SlotInfo
+	// info for store hosted snaps
+	Store *StoreInfo
```

Comment:
it's used when we report about updates (in the REST api and old snappy list afaik)